jsonnet/components/dashboards.libsonnet

```diff
@@ -49,7 +50,16 @@ function(params)
       false
   ;
 
-  local glib = kubernetesGrafana(cfg) {};
+  local nodeDashboards = [
+    'k8s-resources-node.json',
+    'node-rsrc-use.json',
+  ];
+
+  local glib = kubernetesGrafana(cfg) + {
+    _config+: {
+      clusterLabel: 'cluster',
+    },
+  };
```

Comment:
This is the default cluster label that are added in the dashboards. I should be able to pull this from the cfg structure of on of the mixin imports. So far I have failed to find the right config though.

Reply:
Sorry if I'm missing something but in `github.com/brancz/kubernetes-grafana/grafana/grafana.libsonnet` I don't see this `clusterLabel` defined so I'm not sure if I'm missing something or you meant to pull it from somewhere else 🤔

Reply:
The default config in question lives here: https://github.com/kubernetes-monitoring/kubernetes-mixin/blob/07ee0700006ecbc8704ff0b2790fccac18c36c6d/config.libsonnet#L85

Through a convoluted path, we pull that in and I _think_ this should be available in our `cfg` object.

Reply:
Thanks for pointing out where is the value that we want. But AFAICT that mixin is not being included in `dashboards.libsonnet` nor is it being passed in the function pram in `main.libsonnet` so AFAIK it seems that here we would either have to include that jsonnet file that you linked or pass the config in the params, if I understand things correctly

Reply:
yes, that looks perfect. Thank you! :heart: